js/react_native/e2e/android/gradle.properties

```diff
@@ -20,4 +20,7 @@
 android.useAndroidX=true
 android.enableJetifier=true
 FLIPPER_VERSION=0.54.0
-org.gradle.jvmargs=-Xmx4096M
+
+systemProp.javax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
```

Comment:
These classes are for Oracle SDK(Sun SDK). They probably do not exist in Android or iOS. Could you please explain why this change is needed?